client/components/Dashboard.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import AdminPanel from './AdminPanel';
+
```

Comment:
This is the dashboard, once a user is logged in this component is rendered. The dashboard doesn't have much for a standard user, but for an admin it takes care of rendering the `<AdminPanel />` which is where all of the CRUD happens